src/Object/Post.php

```diff
@@ -739,8 +747,7 @@ protected function getParent()
 	/**
 	 * Set conversation thread
 	 *
-	 * @param Thread $thread
-	 *
+	 * @param Thread $thread Thread to set or NULL
```

Comment:
Already inferred from signature.
```suggestion
	 * @param Thread $thread
```